res/css/views/rooms/_IRCLayout.scss

```diff
@@ -178,7 +178,7 @@ $irc-line-height: $font-18px;
         overflow: hidden;
         display: flex;
 
-        > .mx_SenderProfile_displayName {
+        > .mx_SenderProfile_displayName, .mx_SenderProfile_mxid {
             overflow: hidden;
             text-overflow: ellipsis;
             min-width: var(--name-width);
```

Comment:
Hmm, won't this give them each the min-width rather than them sharing it as they did previously? Same might apply to the name column resizer in irc mode

Reply:
Yeah, this doesn't work at all. /me is a total idiot sometimes when trying to do something quickly. I'll look into this tomorrow when I have more time. Making into a draft until then